test/unit/helpers/util/Expect.js

```diff
@@ -197,7 +198,7 @@ define([
         fail('Wait condition not met');
       }
       else {
-        setTimeout(check.bind(null, success, fail, triesLeft - 1), WAIT_INTERVAL);
+        setTimeout(check.bind(null, success, fail, triesLeft - 1), WAIT_MAX_INTERVAL);
```

Comment:
20ms interval wait will fail email tests, so that i increased this to match throttle of 100

Reply:
why it failed? the `wait*` will retry many time with 2000ms.
```
var numTries = (timeout || WAIT_MAX_TIME) / WAIT_INTERVAL;
```

Reply:
i fixed this by `this.handleSubmit = _.throttle(this.handleSubmit, 100, { leading: false });`